drivers/gpio/sabertooth_analog_driver.go

```diff
@@ -0,0 +1,294 @@
+package gpio
+
+import (
+    "errors"
+    "gobot.io/x/gobot"
+    "math"
+)
+
+var (
+    ErrBidirectionalOperation = errors.New("operation possible in bidirectional mode only")
+    ErrA1NotSet = errors.New("pin to driver's terminal A1 must be set")
+    ErrA2NotSet = errors.New("pin to driver's terminal A2 must be set")
+    ErrUnidirectionalCCW = errors.New("pins to driver's terminals A1 and A2 must be set to apply a CCW throttle in unidirectional mode")
+)
+
+// SabertoothDIPSwitches represents the DIP switch positions on the sabertooth driver
+type SabertoothDIPSwitches uint8
+
+// NewSabertoothDIPSwitches creates a new SabertoothDIPSwitches given the
+// position (true: ON, false: OFF) of the DIP switches 3, 4, 5 and 6.
+func NewSabertoothDIPSwitches(powerSupply, mixed, linear, bidirectional bool) SabertoothDIPSwitches {
```

Comment:
An example (/gobot/examples/*) would be nice to understand how to use the dip switches together with the driver.